lib/battleship/game.ex

```diff
@@ -32,86 +35,44 @@ defmodule Battleship.Game do
     {:error, "Game is full"}
   end
 
-  def insert_boat(boat, %{player1: %{boats: boats_list}} = state, pid)
-      when state.player1.pid == pid do
-    # if is_position_valid?(boat, boats_list) do
-    #   state =
-    #     state
-    #     |> put_in([:player1, :boats], state.player1.boats ++ boat)
+  def insert_boat({boat, pid}, %{player1: %{pid: pid1}, player2: %{pid: pid2}, mode: :setting} = state) do
+    {atom, other} = player(pid, pid1, pid2)
```

Comment:
Maybe calling these variables as 'atom' and 'other' is too abstract?

Reply:
I agree with you Sergio. Better if I changed them for example to active_player and other_player.